.github/workflows/ci.yaml

```diff
@@ -0,0 +1,29 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  core:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install YAJL
+      run: sudo apt-get install -y libyajl-dev
+      continue-on-error: true
```

Comment:
I have this job set as continue on error, because GitHub seems to cache the VM for some small period of time. If the VM is reused and YAJL-dev is already installed then this install job (and therefore the build) will fail. On the other, if YAJL is not successfully installed, the config job will fail because YAJL will not be found.